README.md

```diff
@@ -65,30 +65,47 @@ $ ant -Dcluster.config=platform build
 ```
 Cleanup:
 ```
-$ ant -q clean
+$ git clean -Xdf
 ```
 
 #### Notes:
 * You can also use `php`, `enterprise`, etc. See the [cluster.properties](https://github.com/apache/netbeans/blob/master/nbbuild/cluster.properties) file.
 * Once built, you can simply open individual modules of interest with NetBeans and run/rebuild/debug them like any other project
+* The nbbuild folder should contain the portable NetBeans zip
 
 #### Generating Javadoc
 
 Build javadoc:
 ```
-$ ant build javadoc
+$ ant javadoc
 ```
 
-**Note** Run `javadoc-nb` task in Netbeans to run the javadoc build and display it in a web browser.
+#### Notes:
+* On JDK 24 or later, building javadoc may require to rise the jaxp entity limit by setting `export ANT_OPTS=-Djdk.xml.totalEntitySizeLimit=200000`
+* Run `javadoc-nb` task in Netbeans to run the javadoc build and display it in a web browser.
 
 ### Running NetBeans
 
 Run the build:
 ```
 $ ant tryme
 ```
+or
+```
+$ cd nbbuild/netbeans
+$ bin/netbeans
```

Comment:
there were some concerns voiced on slack to advise users to run builds from `nbbuild/netbeans/bin`.

I will rephrase this to advise to extract the zip instead and set `--userdir` to a temp dir if needed.

I personally don't like `ant tryme` since it doesn't run NB in release mode (`fullhack` property etc). It shouldn't be used to verify builds but it is probably ok to keep it in the readme for quick testing.